shared/wallets/send-form/asset-input/container.js

```diff
@@ -4,12 +4,13 @@ import * as WalletsGen from '../../../actions/wallets-gen'
 import {namedConnect} from '../../../util/container'
 import * as Route from '../../../actions/route-tree'
 import * as Constants from '../../../constants/wallets'
+import {anyWaiting} from '../../../constants/waiting'
 
 const mapStateToProps = state => {
   const accountID = state.wallets.selectedAccount
   const currency = state.wallets.building.currency
-
-  const displayUnit = Constants.getCurrencyAndSymbol(state, currency)
+  const currencyWaiting = anyWaiting(state, Constants.loadDisplayCurrencyWaitingKey)
+  const displayUnit = currencyWaiting ? '' : Constants.getCurrencyAndSymbol(state, currency)
```

Comment:
maybe this should blank when `getCurrencyAndSymbol` returns 'XLM'. to avoid the flicker in every time the send form opens